rest/endpoints/user_recordings/profile_recordings.yml

```
@@ -381,6 +381,36 @@ paths:
                     type: array
                     items:
                       $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/General/ForbiddenEntity.yml#/ForbiddenError"
+  # ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+  /most_popular/{period}:
+    get:
+      description: |
+        * **Available since:** BPLAT-12223
+      summary: Get most popular recordings
+      tags:
+        - Recordings
+      operationId: searchMostPopular
```

according to https://github.com/booxmedialtd/frontend/pull/4473/files#diff-cfcba72148aca7654aaac53fe0b4e949R360
```suggestion
      operationId: getMostPopularBroadcastSlots
```